package.json

```
@@ -1,5 +1,9 @@
 {
   "name": "splitwise3",
+  "workspaces": [
+    "./workspaces/api",
+    "./workspaces/web"
```

Let's not change `src` to `workspaces`
```suggestion
"./src/api",
"./src/web"
```

---

alright, let me revert it back